workspace/projects/date-value-accessor/package.json

```diff
@@ -28,8 +28,8 @@
   },
   "homepage": "https://github.com/johanneshoppe/angular-date-value-accessor",
   "peerDependencies": {
-    "@angular/common": "^9.0.0 || ^10.0.0 || ^11.0.0 || ^12.0.0 || ^13.0.0",
-    "@angular/core": "^9.0.0 || ^10.0.0 || ^11.0.0 || ^12.0.0 || ^13.0.0"
+    "@angular/common": ">= 9.0.0 < 15.0.0",
+    "@angular/core": ">= 9.0.0 < 15.0.0"
```

Comment:
Shouldn't we exclude Angular 14.0.0 up to 14.0.x ? Because these versions have s broken Angular CLI?;

Reply:
What do you mean? 🙂

Reply:
We know that because of the double negation bug some paramus won't work. Therefore ngh is not compatible with 14.0.1, 14.0.2, 14.0.3, but maybe already with 14.0.04. we should check in which version the patch had landed

Reply:
But we should better make this constraint on '@angular/cli'

Reply:
This is the date-value-accessor 😉